V.3.0.0 rc1

RikkeVestesen opened the pull request:
OVI will you look at the changes? I am not sure if the version is correct, and I do not know if we will update the IG now (and release it), or if more issues has to be handled before the new version is released.

ovi-medcom commented on input/fsh/MedComMessagingMessageDefinition-fhir.fsh:
```diff
@@ -5,7 +5,45 @@ Description: "An example profile of the MessageDefinition-fhir resource."
 * identifier.system = $MessageDefinitionIdentifierSystem#urn:dk:healthcare:medcom:messaging:fhir (exactly)
 * identifier.type = $MedComMessageDefinitionType#FHIR
 * useContext.code = $UsageContextCodeSystem#workflow (exactly)
+* responseRequired = #always (exactly)
```
Forgot that this is in fact true for all ordinairy messages, but not true for Acknowledgement. responseRequired need just to be a mustSupport element, and the exact code must then be placed in the concrete messagedefinitions

ovi-medcom commented on input/fsh/MedComMessagingMessageDefinition-fhir.fsh:
```diff
@@ -5,7 +5,45 @@ Description: "An example profile of the MessageDefinition-fhir resource."
 * identifier.system = $MessageDefinitionIdentifierSystem#urn:dk:healthcare:medcom:messaging:fhir (exactly)
 * identifier.type = $MedComMessageDefinitionType#FHIR
```
This could be (exactly) too

ovi-medcom commented on input/fsh/MedComMessagingMessageDefinition-fhir.fsh:
```diff
@@ -5,7 +5,45 @@ Description: "An example profile of the MessageDefinition-fhir resource."
+* allowedResponse.message.value = "http://medcomfhir.dk/ig/acknowledgement/ImplementationGuide/medcom.fhir.dk.acknowledgement"
+//InteroperabilityConsultant
+* contact[InteroperabilityConsultant].name = "MedCom FHIR Team"
+* contact[InteroperabilityConsultant].telecom.value = "FHIR@medcom.dk"
```
I would put fhir in lower-case

ovi-medcom reviewed (changes requested):
when changed according to the review comments, the pullrequest may be executed by yourself

RikkeVestesen commented:
@ovi-medcom: I made the corrections: Changed responseRequired to only be MS and correceted a provenance example to not be a CareCommunication example.

RikkeVestesen commented:
@ovi-medcom: I made the corrections: Changed responseRequired to only be MS and correceted a provenance example to not be a CareCommunication example and added (exactly). Should I merge?

ovi-medcom commented:
> @ovi-medcom: I made the corrections: Changed responseRequired to only be MS and correceted a provenance example to not be a CareCommunication example and added (exactly). Should I merge?

Yes :-)
